tests/datastore_pymongo/test_pymongo.py

```diff
@@ -46,163 +50,244 @@ def _exercise_mongo(db):
+    ("Datastore/MongoDB/allOther", 25),
+    ("Datastore/operation/MongoDB/create_index", 1),
+    ("Datastore/operation/MongoDB/find", 3),
+    ("Datastore/operation/MongoDB/find_one", 1),
+    ("Datastore/statement/MongoDB/%s/create_index" % MONGODB_COLLECTION, 1),
+    ("Datastore/statement/MongoDB/%s/find" % MONGODB_COLLECTION, 3),
+    ("Datastore/statement/MongoDB/%s/find_one" % MONGODB_COLLECTION, 1),
+    ("Datastore/operation/MongoDB/bulk_write", 1),
+    ("Datastore/operation/MongoDB/insert_one", 4),
+    ("Datastore/operation/MongoDB/insert_many", 1),
+    ("Datastore/operation/MongoDB/replace_one", 1),
+    ("Datastore/operation/MongoDB/update_one", 1),
+    ("Datastore/operation/MongoDB/delete_one", 1),
+    ("Datastore/operation/MongoDB/delete_many", 1),
+    ("Datastore/operation/MongoDB/find_raw_batches", 1),
+    ("Datastore/operation/MongoDB/create_indexes", 1),
+    ("Datastore/operation/MongoDB/list_indexes", 1),
+    ("Datastore/operation/MongoDB/aggregate", 1),
+    ("Datastore/operation/MongoDB/aggregate_raw_batches", 1),
+    ("Datastore/operation/MongoDB/find_one_and_delete", 1),
+    ("Datastore/operation/MongoDB/find_one_and_replace", 1),
+    ("Datastore/operation/MongoDB/find_one_and_update", 1),
+    ("Datastore/statement/MongoDB/%s/bulk_write" % MONGODB_COLLECTION, 1),
+    ("Datastore/statement/MongoDB/%s/insert_one" % MONGODB_COLLECTION, 4),
+    ("Datastore/statement/MongoDB/%s/insert_many" % MONGODB_COLLECTION, 1),
+    ("Datastore/statement/MongoDB/%s/replace_one" % MONGODB_COLLECTION, 1),
+    ("Datastore/statement/MongoDB/%s/update_one" % MONGODB_COLLECTION, 1),
+    ("Datastore/statement/MongoDB/%s/delete_one" % MONGODB_COLLECTION, 1),
+    ("Datastore/statement/MongoDB/%s/delete_many" % MONGODB_COLLECTION, 1),
+    ("Datastore/statement/MongoDB/%s/find_raw_batches" % MONGODB_COLLECTION, 1),
+    ("Datastore/statement/MongoDB/%s/create_indexes" % MONGODB_COLLECTION, 1),
+    ("Datastore/statement/MongoDB/%s/list_indexes" % MONGODB_COLLECTION, 1),
+    ("Datastore/statement/MongoDB/%s/aggregate" % MONGODB_COLLECTION, 1),
+    ("Datastore/statement/MongoDB/%s/aggregate_raw_batches" % MONGODB_COLLECTION, 1),
+    ("Datastore/statement/MongoDB/%s/find_one_and_delete" % MONGODB_COLLECTION, 1),
+    ("Datastore/statement/MongoDB/%s/find_one_and_replace" % MONGODB_COLLECTION, 1),
+    ("Datastore/statement/MongoDB/%s/find_one_and_update" % MONGODB_COLLECTION, 1),
+]
+
+
+def test_mongodb_client_operation():
+    if pymongo.version_tuple < (4, 0):
+        _test_pymongo_client_scoped_metrics = _test_pymongo_scoped_metrics_v3
+        _test_pymongo_client_rollup_metrics = _test_pymongo_rollup_metrics_v3
+    else:
+        _test_pymongo_client_scoped_metrics = _test_pymongo_scoped_metrics_v4
+        _test_pymongo_client_rollup_metrics = _test_pymongo_rollup_metrics_v4
+
+    # Add MongoClient metric
+
+    _test_pymongo_client_scoped_metrics.extend([("Function/pymongo.mongo_client:MongoClient.__init__", 1)])
```

Comment:
The below extends should be moved into the `_test_pymongo_scoped_metrics_<version>` and `_test_pymongo_rollup_metrics_<version>` for consistency.